### ISSUE TITLE
Include a copy of thosakwe/scripts LICENSE

### ISSUE THREAD

**Issue by thosakwe** [rbcprolabs/dpm#2]
As per the MIT license (https://github.com/thosakwe/dart_scripts/blob/master/LICENSE):

```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```

It is necessary to include a copy of the original license/notice.

I don't do anything MIT-licensed anymore, but since `scripts` does have the MIT license, you don't need to do anything else to be compliant with it.

You won't need to include a notice in each file - just append it your `LICENSE` file and all will be well.

Cheers!

**Comment by SergeShkurko** [comment]
Thanks

**Comment by thosakwe** [comment]
The LICENSE file should contain all text found here: https://github.com/thosakwe/dart_scripts/blob/master/LICENSE

**Comment by SergeShkurko** [comment]
@thosakwe, sorry, solved in #3

**Comment by thosakwe** [comment]
I should clarify - you don't have to remove your name from the license. What I'm saying is that the sections that were written by me should include that license. Whether it comes down to including a copy of the original license in the LICENSE file (along with whatever license you are setting `dpm` to), or comments in the Dart source above the parts taken from `thosakwe/scripts`.
